Approved/implicitly-scoped-qubit-allocation.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Implicitly-scoped qubit allocation
+description: Provides more convenient syntax for allocating qubits.
+author: Sarah Marshall
+date: October 27, 2020
+---
+
+# Proposal
+
+1. Deprecate `using` and `borrowing` keywords and replace with `use` and `borrow`.
+2. Allow `use` and `borrow` statements to be followed by either a block `{ ... }` or a semicolon `;`.
+3. Deprecate and remove parentheses around block statement headers.
```

Comment:
Is this something that was decided on based on user data? I personally find the parentheses helpful in quickly parsing the code while looking through it.

Reply:
I'd like to keep this conversation in the other thread that is also about removing parentheses.